openwisp_network_topology/tests/static/group.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
please move all the test files for this feature into a dedicated directory with an easily recognizable name